### PR TITLE
Continue when glTF2 lights fail to parse.

### PR DESCRIPTION
**Pull request by fire**
Bugfix.

### CODE REVIEW

**Comment by Calinou** [comment]
Does this need to be remade for `3.x` too?

**Comment by fire** [comment]
Needs to be backported.

File: modules/gltf/gltf_document.cpp [hunk 1]
```diff
@@ -30,6 +30,7 @@
 
 #include "gltf_document.h"
 
+#include "core/error/error_macros.h"
```
Comment (akien-mga) [inline modules/gltf/gltf_document.cpp:33]: Spot the potential issue :)
Screenshot_20210809_083230

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.4.

**Pull request merged**